apps/documentation/src/styles.css

```diff
@@ -18,13 +18,25 @@
 
   --primary-color: #e90364;
   --accent-color: #fa2c05;
+  --dark-shadow: 0 0 0 1px rgba(0, 0, 0, 0.1), 0 1px 2px rgba(0, 0, 0, 0.2);
 }
 
 .dark {
   color-scheme: dark;
 
   --primary-color: #fc2781;
   --accent-color: #fb5637;
+
+  --docsearch-modal-background: theme('colors.zinc.900');
+  --docsearch-modal-shadow: var(--dark-shadow);
+  --docsearch-searchbox-focus-background: theme('colors.zinc.800');
+  --docsearch-searchbox-shadow: var(--dark-shadow);
+  --docsearch-footer-background: theme('colors.zinc.900');
+  --docsearch-footer-shadow: var(--dark-shadow);
+  --docsearch-text-color: white;
+  --docsearch-hit-background: theme('colors.zinc.800');
+  --docsearch-hit-shadow: var(--dark-shadow);
+  --docsearch-hit-color: white;
```

Comment:
Can you please also add the variables for the keys, etc.? Just noticed that they were missing from your screenshot in the issue.

Full list: https://github.com/algolia/docsearch/blob/main/packages/docsearch-css/src/_variables.css#L62-L69

Missing variables:
```
--docsearch-container-background
--docsearch-searchbox-background
--docsearch-key-gradient
--docsearch-key-shadow
--docsearch-logo-color
--docsearch-muted-color
```

Reply:
fwiw, to me, only the keys are noticeable as "odd" and "out of place". We may not need to override the rest, so please use your discretion.

Reply:
@benjaminforras sorry for the repeated comments 😅 I just noticed that the dark theme already exists and can be enabled by adding `data-theme="dark"` to the `html` tag. This might be sufficient, but wouldn't match the rest of the zinc-based theme so I do think we should go with your approach.

Reply:
![image](https://github.com/user-attachments/assets/898e9938-c00d-4a1f-96ec-71862244454b)

No problem. I've reviewed the dark mode implementation from the link you provided, and I believe my adjustments, as shown in the image above, offer a more cohesive and visually appealing experience. I think this approach might be more effective than the default dark mode styling in the Algolia docs. Let me know your thoughts.

Reply:
> @Abodawoud sorry for the repeated comments 😅 I just noticed that the dark theme already exists and can be enabled by adding `data-theme="dark"` to the `html` tag. This might be sufficient, but wouldn't match the rest of the zinc-based theme so I do think we should go with your approach.
> 
> Edit: @benjaminforras sorry, didn't mean to tag you 😅

No problem I am just learning from you guys that is my first contribution😂❤️

Reply:
This looks great!